man/labgrid-client.1

```diff
@@ -88,6 +92,11 @@ for the \fB+\fP place expansion.
 This variable can be used to specify a state which the device transitions into
 before executing a command. Requires a configuration file and a Strategy
 specified for the device.
+.SS LG_INITIAL_STATE
+.sp
+This variable can be used to specify an initial state the device is known to
+be in. This is useful during development. The Strategy used must implement the
+\fBforce()\fP method. Requires a desired state.
```

Comment:
What do you mean with 'Requires a desired state'? In addition to the `force()` method, it has the same requirements as LG_STATE (environment and strategy).

Reply:
"Desired state" means -s/--state. Any suggestions how to improve the wording?